cpp/throwaways/extract_both_700_and_71X.cc

```diff
@@ -0,0 +1,66 @@
+/** \brief Extracts list of PPN that contain both 700 and 710 and 711 fields\n
+ *  \author Johannes Riedl (johannes.riedl@uni-tuebingen.de)
+ *
+ *  \copyright 2020 Universitätsbibliothek Tübingen.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
```

Comment:
Too many blank lines.  We're inconsistent here and have files with zero or one lines here but never with two.  Pick whatever you find more aesthetically pleasing.  (I prefer one blank line.)